src/prebid.js

```diff
@@ -100,6 +100,16 @@ function validateSizes(sizes, targLength) {
   return cleanSizes;
 }
 
+function setBattrForAdUnit(adUnit, mediaType) {
+  const battr = adUnit.ortb2Imp?.[mediaType]?.battr || adUnit.mediaType?.[mediaType]?.battr;
+  if (adUnit.ortb2Imp?.[mediaType] && battr) {
+    adUnit.ortb2Imp[mediaType].battr = battr;
+  }
+  if (adUnit.mediaTypes?.[mediaType] && battr) {
+    adUnit.mediaTypes[mediaType].battr = battr;
+  }
```

Comment:
IMO this should always copy it over, since the goal is to make it easy to find for adapters. For example if I have `{mediaTypes: {video: {battr}}, ortbImp: {}}`, it would remain relegated to `mediaTypes`; turn it around and it would remain only in `ortb2Imp`, and adapters would still need to check both places. (They're also both decent test cases).  We should also probably log a warning if they're both specified but they differ.